### PR TITLE
RSG-10 to be merged into develop

### PR DESCRIPTION
**Pull request by yehorchankov**

### CODE REVIEW

File: src/BLL/Concrete/EmailSender.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                return false;
+            }
+            return true;
+        }
+
+        private EmailCredential GetCredentialsFromConfig()
+        {
+            EmailCredential credential = new EmailCredential
+            {
+                Email = "rp16.serious.games@gmail.com",
```
Comment (askliar) [inline src/BLL/Concrete/EmailSender.cs:52]: @GasperPaul @yehorchankov  Should we leave hardcoded or should it be in some `.config` file? 
Or leave it as it is for now and then change if needed?

**Comment by askliar** [comment]
@yehorchankov Have you already merged Yevgen's branch into yours?

File: src/BLL/Concrete/EmailSender.cs (hunk 1 above)
Comment (GasperPaul) [inline src/BLL/Concrete/EmailSender.cs:52]: @emohat1r @yehorchankov This should be in config. But it's not crucial for tommorow's release.

**Comment by GasperPaul** [comment]
@emohat1r @yehorchankov [He did.](https://github.com/BU-git/rp16_serious_game/network)

File: src/BLL/Concrete/EmailSender.cs (hunk 1 above)
Comment (yehorchankov) [inline src/BLL/Concrete/EmailSender.cs:52]: @emohat1r Ok, I'll do it later

**Comment by yehorchankov** [comment]
@emohat1r I have already merged RSG-8 and RSG-12

**Pull request merged**